cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/parser/YamlParser.java

```diff
@@ -39,7 +41,15 @@ public ParsedSlang parse(SlangSource source) {
             parsedSlang.setName(source.getName());
             return parsedSlang;
         } catch (Throwable e) {
-            throw new RuntimeException("There was a problem parsing the YAML source: " + source.getName() + ".\n" + e.getMessage(), e);
+            String errorMessage = e.getMessage();
+            if (e instanceof ScannerException && errorMessage.startsWith("mapping values")){
+                errorMessage += "Probably did not provide (key: value) pair or missing space after colon(:)";
+            }
+            else if (e instanceof ConstructorException && errorMessage.startsWith("Cannot create property")){
+                errorMessage += "Are you sure you are not missing a space after colon(:) for one of the imports and that all imports have an alias?";
```

Comment:
The exception message starts with: *Cannot create property=imports* and I think you should also the imports text when checking, since you only address errors in imports...